server/src/watchdog.js

```diff
@@ -240,10 +240,12 @@ exports.handleResult = function(req) {
 };
 
 function handleNegative(record) {
+  mozlog.debug("watchdog-negative-match", {msg: `Watchdog submission ${record.id} is a negative match.`});
   return db.update("UPDATE watchdog_submissions SET positive_result = FALSE WHERE id = $1", [record.id]);
 }
 
 function handlePositive(record) {
+  mozlog.warn("watchdog-positive-match", {msg: `Watchdog submission ${record.id} is a positive match.`});
   return db.transaction(client => {
     return db.queryWithClient(
       client,
```

Comment:
Suggestion: since we're in here anyway, it might be useful to log the outcome of the transaction.

Maybe something like:
```js
return db.transaction(...).then(() => { 
  mozlog.info(`Watchdog positive match ${record.id} successfully committed to the database.`)
}).catch(err => {
  mozlog.error(`Watchdog positive match ${record.id} not committed to the database: ${err}.`);
  throw err;
})
```